CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2021, 鍾淯丞, 周杰仕, 林仁鴻. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+cmake_minimum_required(VERSION 3.20)
+project(TestRequestParser)
+enable_testing()
+
+set (CMAKE_CXX_STANDARD 14)
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/cmake/Modules/")
+
+find_package(GTest REQUIRED)
+
+include_directories(${GMOCK_ROOT}/include)
+include_directories(${GMOCK_ROOT}/gtest/include)
+include_directories(${GTEST_INCLUDE_DIRS})
+
+include_directories(./src/Header)
+
+#will be used when other branch is merged
+#add_executable(TestRequestParser 
+#		src/Implement/UnitTestRequestParser.cpp
+#		src/Implement/RequestParser.cpp
+#		src/Header/RequestParser.h
+#		src/Header/OperateData.h
+#		)
+
+#target_link_libraries(TestRequestParser pthread GTest::gtest GTest::gtest_main GTest::gmock GTest::gmock_main)
```

Comment:
[Optional] Consider moving pthread to an if block that is restricted to *nix.

Reply:
Done